extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -152,12 +161,35 @@ const renderOutlinkPageAttachmentUI = (
     openAttachmentEl.setAttribute('href', attachmentHref);
   }
 
-  // Setting theme
-  openAttachmentEl.setAttribute('theme', attachmentEl.getAttribute('theme'));
-
   // Getting elements
   const {chipEl, ctaLabelEl} = htmlRefs(openAttachmentEl);
 
+  // Setting theme
+  const themeAttribute = attachmentEl.getAttribute('theme');
+  openAttachmentEl.setAttribute('theme', themeAttribute);
+  let labelTextColor = themeAttribute === 'dark' ? 'white' : 'black';
+
+  const accentColor = attachmentEl.getAttribute('cta-accent-color');
```

Comment:
Nit: If the accent color is only used inside the `if` block, we can move it inside there and avoid running unnecessary lines of code. If we use specifying the accent color as the opt-in to the custom theme, it makes sense to declare this variable outside though (and then do `if (accentColor) {// theme is custom}`)

Reply:
Moved it into the if block